RecoLocalTracker/SiStripClusterizer/plugins/ClustersFromRawProducer.cc

```diff
@@ -355,9 +355,28 @@ try {
 	  }                                               
 	  continue;
 	}
-      } else {
+    } else if likely(mode == sistrip::READOUT_MODE_ZERO_SUPPRESSED_LITE8 ) {
```

Comment:
I left these two separated as I wanted to start using the static ZS lite unpacker defined in the SiStrip BSUnpacker (BS = bit stripping), which can be used for basically any word ADC length (here, 8-bit). Unfortunately the time constraints prevented me to test it so far...
So I guess I can collapse these as from now.